firefox/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "OctoLinker",
+  "version": "4.1.1",
```

Comment:
Oh, ok. Maybe we're thinking about these version numbers differently. I'll go ahead and merge this PR, then continue the conversation in #101